BankAccount/pkg/store/transactionhistory.go

```diff
@@ -1 +1,72 @@
 package store
+
+import (
+	"bankacc/pkg/entities"
+	"database/sql"
+)
+
+type TransactionHistoryStore interface {
+	Insert(userId int, accountId int, amount float64, action string, createdAt string) (*entities.TransactionHistory, error)
+	GetTransactionsById(id int) (*[]entities.TransactionHistory, error)
+	GetTransactionsByIdFromToDate(id int, fromDate string, toDate string) (*[]entities.TransactionHistory, error)
```

Comment:
@vpatcev18 fromDate and toDate should be `time.Time` instead of a string.

Reply:
@FilipNikolovski Resolved types